furimoe.alicemagiclauncher.json

```diff
@@ -3,10 +3,11 @@
   "providerName": "Furimoe",
   "serviceId": "alicemagiclauncher",
   "serviceName": "Alice Magic Launcher DNS TXT",
-  "version": 3,
+  "version": 1,
   "description": "Instance manages DNS TXT records for Alice Magic Launcher.",
   "syncPubKeyDomain": "keys.furimoe.com",
   "syncRedirectDomain": "furimoe.com",
+  "syncBlock": false,
```

Comment:
For booleans default is false, that takes effect for instance when a key is not mentioned.

Also, version going from 3 to 1 is a bit strange.